### DISCUSSION TITLE
[Discussion] about  parser with eslint

### DISCUSSION THREAD

**Discussion by richerfu** [oxc-project/oxc#306] [General]
Is it possible to create a package for eslint parser, just like `typescript-eslint/parser`.
I am not sure that it can or not get more better effects  and i find some issue with eslint about this.

https://github.com/swc-project/swc/issues/246

**Comment by richerfu** [comment]
I think it's important for us to be compatible with eslint ecosystem without a new tools. Just like `rspack` is to `webpack`

**Comment by Boshen** [comment]
Thank you for asking. This is a really big question, the answer is related to your familiarity with the ESLint, JavaScript ecosystem, Rust <-> JS interop etc

And FYI, I have personally asked the author of ESLint on this subject, and he is fine with this project.

**Reply by richerfu** [reply]
Cool. Is there any planning?

**Reply by Boshen** [reply]
Unfortunately no, I don't have the bandwidth to maintain a compatibility layer just for eslint.

**Reply by bacarybruno** [reply]
Hello! Would you be open to external contribution on this topic?
